entry_point_test.go

```diff
@@ -17,28 +17,49 @@
 package test
 
 import (
+	"os"
 	"testing"
 
 	"github.com/stretchr/testify/suite"
 
 	"github.com/networkservicemesh/integration-tests/suites/basic"
 	"github.com/networkservicemesh/integration-tests/suites/features"
 	"github.com/networkservicemesh/integration-tests/suites/heal"
+	"github.com/networkservicemesh/integration-tests/suites/interdomain"
 	"github.com/networkservicemesh/integration-tests/suites/memory"
 )
 
 func TestRunHealSuite(t *testing.T) {
+	if os.Getenv("INTERDOMAIN-CI") == "enabled" {
+		t.Skip()
+	}
 	suite.Run(t, new(heal.Suite))
 }
 
 func TestRunFeatureSuite(t *testing.T) {
+	if os.Getenv("INTERDOMAIN-CI") == "enabled" {
+		t.Skip()
+	}
```

Comment:
## Option 1
Consider using build tags.

Note: you can separate tests into two files with two tags
```go
// file1.go

// +build interdomain
```
...
```go
// file2. go

// +build single
```
And running of test

```go
go test -tags=interdomain
go test -tags=single
```

See at https://pkg.go.dev/go/build.


## Option 2

Run tests via Match (where names of test should contain "Interdomain" or "Single", for example, "Tes1Single", "Test2Interdomain")
  
```
go test -run Intedomain
go test -run Signle
```

Or via pkg
```
go test pkg/single
go test pkg/interdomain
```

Reply:
ok, done
used build tags